### ISSUE TITLE
Bug ? Grève des profs

### ISSUE THREAD

**Issue by Laltar** [JulienNigon/SpellcasterUniversity-Bug-Report#647]
Bonjour,

Accroc à votre jeu depuis deux jours, j'ai remarqué un chose lorsque je n'ai pas les moyens d'offrir une cafetière à mes chers professeurs, la grève qui est censé durer 400 sec, s'éternise ad vitam eternam, ce qui bloque toute ma progression lorsque celle ci intervient en début de partie.

J'ai un deuxième point qui est plus un questionnement, je trouve la gestion des besoins des professeurs extrêmement difficile, leurs jauges se remplissent en effet de manière instantanées après un certain temps. J'ai beaucoup de mal a endigué cela même en multipliant les salles des profs et les appartements privés.

Bonne continuation, dans l'attente de former mon premier archimage, merci pour ce jeu.

**Comment by allikrid** [comment]
Exact. La grève a commencé année 4 et était encore là année 17 pour moi.
